faketests/slowtests/test_1.py

```diff
@@ -0,0 +1,33 @@
+import time
```

Comment:
TIL!

Once it was easy to manipulate, I settled on 7 tests each. Having the test count not be divisible by the number of cores makes sure they spread out well for the best visualization.